File: Estrutura_Repetição/Python/Ex1072.py
quant = int(input())


dentro = 0
fora = 0
for i in range(quant):
    v = int(input())

    if 10 <= v <= 20:
        dentro += 1
    else:
        fora += 1

print(f"""{dentro} in
{fora} out""")

File: Estrutura_Repetição/Clojure/Ex1072.clj
(defn contar-valores [n]
  (let [valores (repeatedly n #(Integer/parseInt (read-line)))]
    (let [dentro (reduce + (map #(if (and (>= % 10) (<= % 20)) 1 0) valores))
          fora (- n dentro)]
      (println (str dentro " in"))
      (println (str fora " out")))))

(defn main []
  (let [n (Integer/parseInt (read-line))]
    (contar-valores n)))

(main)